custom-views-templates/starter-typescript/src/components/entry-point/entry-point.tsx

```diff
@@ -0,0 +1,27 @@
+import { lazy } from 'react';
+import {
+  CustomViewShell,
+  setupGlobalErrorListener,
+} from '@commercetools-frontend/application-shell';
+import loadMessages from '../../load-messages';
+
+// Here we split up the main (app) bundle with the actual application business logic.
+// Splitting by route is usually recommended and you can potentially have a splitting
+// point for each route. More info at https://reactjs.org/docs/code-splitting.html
+const AsyncApplicationRoutes = lazy(
+  () => import('../../routes' /* webpackChunkName: "routes" */)
+);
+
+// Ensure to setup the global error listener before any React component renders
+// in order to catch possible errors on rendering/mounting.
+setupGlobalErrorListener();
+
+const EntryPoint = () => (
+  <CustomViewShell applicationMessages={loadMessages}>
+    <AsyncApplicationRoutes />
+  </CustomViewShell>
```

Comment:
From a user perspective this is now essentially the same as rendering `<ApplicationShell>` (simple and familiar).